frontend/src/pages/ReservationCalendar.tsx

```diff
@@ -33,22 +33,43 @@ function ReservationCalendar() {
     }
   };
 
+  const mergeTimeloSlots = (timeslots: EventInput[]): EventInput[] => {
+    let mergedTimeslots: EventInput[] = [];
+
+    timeslots.forEach((timeslot) => {
```

Comment:
kokeilin muokata timeslottia ja sen jälkeen tää ei toiminut. Järjestys ilmeisesti muuttuu, joten tässä pitäis sortata timeslotit